include/picongpu/simulation/stage/CurrentInterpolationAndAdditionToEMF.hpp

```diff
@@ -95,14 +96,14 @@ namespace picongpu
                  * @param step index of time iteration
                  * @param fieldSolver field solver
                  */
-                void operator()(uint32_t const step, fields::Solver& fieldSolver) const
+                void operator()(uint32_t const step, fields::Solver& fieldSolver, bool allowCommunication) const
```

Comment:
I don't see the reason to introduce this variable, as you set it to always `true` in `include/picongpu/simulation/control/Simulation.hpp` anyways.

Reply:
@psychocoderHPC Could you explain, why you want to have the option to disable communication?

Reply:
I can remove this from this PR. I overlooked this change, it is required for the StaticCurrentDistribution feature.
For StaticCurrentDistribution we need to communicate the fields only once depending on the INterpolation stencil width.

Reply:
I removed the changes from this PR and will introduce it together with the new feature.